src/tsf/TfDispAttr.cpp

```diff
@@ -54,7 +51,7 @@ HRESULT CicDisplayAttributeMgr::GetDisplayAttributeTrackPropertyRange(TfEditCook
     HRESULT hr = E_FAIL;
 
     ULONG ulNumProp;
-    ulNumProp = (ULONG) m_DispAttrProp.Count();
+    ulNumProp = (ULONG) m_DispAttrProp.size();
     if (ulNumProp) {
         const GUID **ppguidProp;

```

Comment:
Should we use unique_ptr here to prevent manually `delete`?

Reply:
for ppguidProp? Yeah that's probably wise. You're welcome to clean up further than just the ATL requirements while you're poking about here.

Reply:
Added try blocks and changed ppguidProp to unique_ptr.